libs/schemas/src/entities/notification.schemas.ts

```diff
@@ -40,8 +43,33 @@ export const ThreadSubscription = z.object({
   id: PG_INT.optional(),
   user_id: PG_INT,
   thread_id: PG_INT,
-  created_at: z.date().optional(),
-  updated_at: z.date().optional(),
+  created_at: z.coerce.date().optional(),
+  updated_at: z.coerce.date().optional(),
```

Comment:
do we need to coerce here? all other `lib/schemas/src/entities` seem to just use `z.date()...`